README.md

```diff
@@ -324,10 +324,19 @@ Table component supporting sorting and filtering.
 | data (required) | Array | The array of data items, each corresponding to a single potential table row | None
 | filter (optional) | Function | Called with data for a single row. Should return `false` if the row should be filtered out, or `true` otherwise. | None
 | fixed (optional) | Boolean | Whether or not table column widths should be fixed (vs fluid). | False
+| initialPage (optional) | Number | The initial page to be displayed initially. | None
 | initialSortState (optional) | `{columnID: String, direction: Table.sortDirection}` | The initial sort state of the table. | None
+| onPageChange (optional) | Function | Callback function for the displayed page change event. | None
 | onSortChange (optional) | Function | Callback function for the sort state change event. | None
+| pageSize (optional) | Number | The number of data rows to display on each page. | 10
+| paginated (optional) | Boolean | Whether or not to enable pagination. See `pageSize` | false
 | rowIDFn (required) | Function | Called with data for a single row. Should return the unique ID for that row. | None
 
+**API**
+
+- `setCurrentPage(page: Number)` - Explicitly sets the displayed page on the Table. Useful for reacting to data or filter changes that warrant resetting the current page.
```

Comment:
And if you want to make it a bit less error prone you can add bounds checking so that if someone does try to use page index 0, it would warn the user that that's invalid; you can implement that as a custom proptype